dhall/dhall.cabal

```
@@ -563,6 +563,7 @@ Test-Suite tasty
         deepseq                   >= 1.2.0.1  && < 1.5 ,
         dhall                                          ,
         directory                                      ,
+        either                    >= 5        && < 5.1 ,
```

One more small suggestion: Since we already define bounds for this dependency in the library component, it's better to leave them off here.
```suggestion
        either                                         ,
```